lib/hammer_cli_foreman/user_mail_notification.rb

```diff
@@ -0,0 +1,54 @@
+module HammerCLIForeman
+
+  class UserMailNotification < HammerCLIForeman::Command
+    resource :mail_notifications
+    command_name 'mail-notification'
+
+    class AddMailNotificationCommand < HammerCLIForeman::CreateCommand
+      action :create
```

Comment:
Sorry, forgot to mention: if you inherit from `[Create|Update|Delete|Info|List]Command` you don't need to specify the `action` _unless_ it's different from default.